keps/prod-readiness/sig-network/1880.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 1880
 alpha:
   approver: "@johnbelamaric"
+beta:
```

Comment:
Switch this one over to me, I'll be taking this over from John, who has quite a few on his board.

Reply:
sure